esphome/components/mitsubishi_uart/muart_packet.h

```diff
@@ -206,6 +206,8 @@ class SettingsGetResponsePacket : public Packet {
 
   float getTargetTemp() const;
 
+  bool getiSeeStatus() const;
```

Comment:
If this is an on/off setting, let's rename this something like getiSeeEnabled or getiSeeOn or the like.